index.js

```diff
@@ -4,18 +4,21 @@ module.exports = postcss.plugin('postcss-banner', function configure(opts) {
   opts = opts || {};
 
   function process(value) {
-    var text = String(value);
-    var comment = text;
-
-    if (!opts.inline) {
-      comment = text.split('\n')
-        .join('\n * ')
-        .concat('\n ');
+    var comment = String(value);
+    var bang = opts.important ? '!' : '';
+
+    if (!!opts.inline) {
```

Comment:
well it is actually good practice in order to make sure we get back a boolean value. !! applied to a value will negate the variable first and then negate again, this way wa always have `true/false` back